.packit.yaml

```diff
@@ -43,6 +43,9 @@ jobs:
       distros: [centos-7, oraclelinux-7]
     epel-8-x86_64:
       # TODO add oraclelinux-8.6 when the 8.7 is officially out
+      # To have a baseline, when the 8.7 Oracle Linux is officially announced,
+      # add the oraclelinux-8.6 target temporarily until we get the official 8.7 image
+      # to the Testing Farm
```

Comment:
Should we add the ol8.6 here because the https://github.com/oamg/convert2rhel/pull/667 adds too? I mean, it will cause a merge conflict, anyway.

Reply:
Makes sense, will do